tests/integration-service/integration.go

```diff
@@ -0,0 +1,160 @@
+                        if err != nil {
+                                if errors.IsForbidden(err) {
+                                        klog.Infof("don't have enough permissions to get a configmap with default pipeline in %s namespace\n", constants.BuildPipelinesConfigMapDefaultNamespace)
+                                } else {
+                                        Fail(fmt.Sprintf("error occured when trying to get configmap %s in %s namespace: %v", constants.BuildPipelinesConfigMapName, constants.BuildPipelinesConfigMapDefaultNamespace, err))
+                                }
+			}
+                        _ = defaultBundleConfigMap.Data["default_build_bundle"]
+
+		})
+		It("triggers a PipelineRun", func() {
+                        Eventually(func() bool {
+                                pipelineRun, err := f.HasController.GetComponentPipelineRun(componentName, applicationName, appStudioE2EApplicationsNamespace, false)
+                                if err != nil {
+                                        klog.Infoln("PipelineRun has not been created yet")
+                                        return false
+                                }
+                                return pipelineRun.HasStarted()
+                        }, timeout, interval).Should(BeTrue(), "timed out when waiting for the PipelineRun to start")
+                })
+
+                When("the PipelineRun has started", Label("integration-service"), func() {
+                        BeforeAll(func() {
+                                timeout = time.Second * 600
+                                interval = time.Second * 10
+                        })
+                        It("should eventually finish successfully", func() {
+                                Eventually(func() bool {
+                                        pipelineRun, err := f.HasController.GetComponentPipelineRun(componentName, applicationName, appStudioE2EApplicationsNamespace, false)
+					Expect(err).ShouldNot(HaveOccurred())
+
+                                        for _, condition := range pipelineRun.Status.Conditions {
+                                                klog.Infof("PipelineRun %s Status.Conditions.Reason: %s\n", pipelineRun.Name, condition.Reason)
+
+                                                if condition.Reason == "Failed" {
+                                                        Fail(fmt.Sprintf("Pipelinerun %s has failed", pipelineRun.Name))
+                                                }
+                                        }
+                                        return pipelineRun.IsDone()
+                                }, timeout, interval).Should(BeTrue(), "timed out when waiting for the PipelineRun to finish")
+                        })
+                })
+
+	//	It("check if the ApplicationSnapshot are created",func() {
```

Comment:
Could we need those two condition present even though they are commented?

Reply:
Yes. It's for future usage. Just leave the specific integration-service functions  to next PR since the e2e-tests checking is not easy to get passed.